packages/core/src/read/readRegistry.js

```diff
@@ -44,8 +44,15 @@ export function get(pattern: ReadDef): ReadFn  {
   return registry.get(pattern);
 }
 
-export function httpRead(url: string): Promise<ReadResponse> {
-  return fetch(url)
+export function httpRead(url: string, revalidate: boolean): Promise<ReadResponse> {
+  const options = {};
+  if (revalidate) {
+    const headers = {
+      'Cache-Control': 'max-age=0'
+    };
+    options['headers'] = headers;
+  }
+  return fetch(url, options)
```

Comment:
Alternative, I could write it like so:
```javascript
export function httpRead(url, revalidate) {
  const headers = new Headers()
  if (revalidate) {
    headers.append('Cache-Control', 'max-age=0')
  }
  const options = { headers }
  return fetch(url, options)
    .then(processResponse)
    .catch(errorResponseForUrl(url))
}
```
But, in that case, if I try to write test that passes here, it will fail (`Headers undefined`).